### DISCUSSION TITLE
Is this Libary still alive?

### DISCUSSION THREAD

**Discussion by telion2** [CodeDredd/pinia-orm#1761] [General]
Last Release was on in September of last year. 
There are multiple Issues reported. 
I am not sure if I should be using pinia-orm if nobody works on it. I also have a few Issues of my own but I need some confidence, that this Library is actually replacing vuex orm. 

Mainly they revolve around being able to completely do CRUD on the Models and Relationships remotely, as well as more flexibility when implementing stuff. 

But I am not sure if I should invest into pinia-orm, if its become stale.

**Comment by CodeDredd** [comment]
Hey @telion2 ,

yes this libary is still alive. I just didn't had the time to make something for it because i moved to a new location and i had to handle a lot of personal stuff. Having kids doesn't make it easier but worth every minute 😉 

Now that almost everything important is done (Unpacking the final boxes) i am going to spend some time with my libraries again. So stay tuned and sorry for the break without notice.

**Comment by telion2** [comment]
That is great to hear. Wish you the best for settling in. 
Sorry if I pressured to much, I had to many libraries that discontinued.

**Reply by CodeDredd** [reply]
It's all fine. 😄  I look it that way. vuex-orm has 128 issues open and i got 25 after 3 months break and mostly feature & doc requests.
And vuex-orm is still today very used although it is stale for so long now. I think the critical point is where a library doesn't keep with the next version it is depending on. But if it is stable and almost bug free then you can confidentanty use it. And at the end it's always possible to make a fork. 

Btw. if you want to make some pressure for me then spend me cup of coffee ^^. Hehehe...i always then have more the urge to do something. 😉 And besides that it's nice to see that the community here is also growing ppl like the "orm" way.

**Comment by Graphmaxer** [comment]
Hello @CodeDredd, I hope everything is okay on your side. It has been a few months without news, are you able to check the pending issues and pull requests ?

**Reply by CodeDredd** [reply]
Look answer down 😉

**Comment by idc77** [comment]
I don't want to be negative, I had a feeling when it was forked, this project is big, a lot of work.
Maybe think about searching for people who'd like to pick up this project?

**Reply by CodeDredd** [reply]
I am already in contact with some1 who wanna join to be maintainer. Sorry for not having spent time on the library the months latly. Moving to new location with kids was more prio and time consuming then i thought. Thought had soon time....but was wrong. ^^
I have not forgotten you all. Today i am gonna do something.
Thanks for your patients and understanding. Keep making PRs or if you wanna motivate me to do more, buy me a coffee 😄.
